pkg/alerts/prometheus.go

```diff
@@ -8,7 +8,7 @@ import (
 	"time"
 
 	"github.com/prometheus/client_golang/api"
-	v1 "github.com/prometheus/client_golang/api/prometheus/v1"
+	"github.com/prometheus/client_golang/api/prometheus/v1"
```

Comment:
goimports made it v1. While it's not necessary, I think it's okay to keep it. Else someone else will land on this and will change it by mistake if using goimports. See for the reason behind v1 aliasing here : https://github.com/golang/go/issues/29041

Reply:
Thanks for investigating.  I guess if we know the reason it's ok to keep it.